docker-compose.yml

```
@@ -29,8 +33,30 @@ services:
       - ./media_volume:/var/html/media/
     ports:
       - "8000:8000"
+    networks:
+      - app_net
     depends_on:
       - backend
 
+  s3:
+    image: minio/minio:latest
+    container_name: s3
+    command: server --console-address ":9001" /data/
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      MINIO_ROOT_USER: ${MINIO_STORAGE_ACCESS_KEY}
+      MINIO_ROOT_PASSWORD: ${MINIO_STORAGE_SECRET_KEY}
```

для общего нейминга, нужно заменить на s3

```suggestion
      MINIO_ROOT_USER: ${S3_ACCESS_KEY}
      MINIO_ROOT_PASSWORD: ${S3_SECRET_KEY}
```